node/full_test.go

```diff
@@ -105,39 +97,6 @@ func TestFull_P2P_Streams(t *testing.T) {
 	require.NoError(t, stream.Close())
 }
 
-func TestFull_RPCServer(t *testing.T) {
-	repo := MockRepository(t, DefaultConfig(Full))
-	node, err := New(Full, repo)
-	require.NoError(t, err)
-	require.NotNil(t, node)
-	require.NotNil(t, node.Host)
-
-	ctx, cancel := context.WithCancel(context.Background())
-	t.Cleanup(cancel)
-
-	err = node.Start(ctx)
-	require.NoError(t, err)
-
-	// register simple ping handler on node's server
-	err = node.RegisterAPI("/ping", ping{})
-	require.NoError(t, err)
-	// send ping
-	resp, err := http.Get(fmt.Sprintf("http://%s/ping", node.Config.RPC.ListenAddr))
-	require.NoError(t, err)
-	// read pong
-	pong, err := ioutil.ReadAll(resp.Body)
-	require.NoError(t, err)
-	t.Cleanup(func() {
-		resp.Body.Close()
-	})
-	assert.Equal(t, "pong", string(pong))
-
-	ctx, cancel = context.WithCancel(context.Background())
-	t.Cleanup(cancel)
-
-	require.NoError(t, node.Stop(ctx))
-}
```

Comment:
Another deleted test? Why? I understand Start/Stop where removed but it would still be good to test that the RPC server runs as expected. Though that could be more of an integration test.

Reply:
We don't use an RPC anywhere. I removed it for now. And it is not an RPC, but just a plain HTTP server. 
#169